draft-ietf-gnap-core-protocol.md

```diff
@@ -5365,3 +5365,693 @@ justification for the data type representing the same basic kind of thing it ext
+            "properties": {
+              "email": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "email"
+            ]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "subject_type": {
+                "const": "phone-number"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "phone_number": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "phone_number"
+            ]
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "subject_type": {
+                "const": "iss-sub"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "iss": {
+                "type": "string"
+              },
+              "sub": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "uri",
+              "sub"
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "type": "object",
+  "properties": {
+    "resources": {
+      "type": [
+        "array",
+        "object"
+      ],
+      "items": {
+        "$ref": "#/definitions/resource"
+      },
+      "additionalProperties": {/* if it's an object for multiple tokens, each field's value is a resource object*/
+        "type": [
+          "array"
+        ],
+        "items": {
+          "$ref": "#/definitions/resource"
+        }
+      }
+    },
+    "subject": {
+      "type": "object",
+      "properties": {
+        "sub_ids": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/subject_id_types"
+          }
+        },
+        "assertions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/assertion_types"
+          }
+        }
+      }
+    },
+    "client": {
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "instance_id": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        },
+        {
+          "type": [
+            "object",
+            "string"
+          ],
+          "properties": {
+            "key": {
+              "$ref": "#/definitions/key"
+            },
+            "class_id": {
+              "type": "string"
+            },
+            "display": {
+              "type": [
+                "object",
+                "string"
+              ],
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "uri": {
+                  "type": "string"
+                },
+                "logo_uri": {
+                  "type": "string"
+                }
+              }
+            },
+            "instance_id": false
```

Comment:
What does this mean?

Reply:
This means `instance_id` is not allowed alongside the other properties.